server/routes/github/github_repo_metadata.py

```
@@ -193,3 +194,131 @@ def generate_repo_metadata(self, user_id: str, repo_full_name: str):
             self.retry(countdown=30)
         except self.MaxRetriesExceededError:
             _update_metadata(user_id, repo_full_name, None, "error")
+
+
+def _import_installation_repos(self, user_id: str, installation_id: int):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_import_installation_repos`

Add return type annotation: `None`

(ANN202)